docs/cli/cmdline/create.md

```diff
@@ -1,71 +1,219 @@
 ---
 title: terramate create - Command
-description: With the terramate create command you can create a new stack in the current project.
+description: Create a new Stack or import existing Terraform or Terragrunt configurations by using the `terramate create` command.
+outline: [2, 3]
 ---
 
-# Create
+# Create or Import Stacks
 
-The `create` command creates a new stack. This command accepts a path that will
-be used to create the stack. The `create` command will create intermediate
-directories as required. Whenever the `create` command is used, the code generation will run automatically to ensure all code relevant for the newly created stack will be generated.
+## Overview
 
-## Usage
+The `terramate create` command creates a new stack in a given directory aka stack path.
+If the stack path does not yet exist, it will be created.
+Code Generation will be triggered by default to initialize the newly created stack.
+
+Stack metadata can be set upon the creation of the stack.
+
+This metadata will also be synchronized to Terramate Cloud if enabled and can help you and your team browse through stacks better and monitor stack health on scale.
 
-`terramate create [options] PATH`
+**The `terramate create` command can also be used to import existing Terraform or Terragrunt configurations and initialize stacks automatically for easy onboarding.**
 
 ## Examples
 
-Create a new stack:
+### Create a new stack
+
+This is the most basic example to create a stack in a directory called `path/to/stack`.
 
 ```bash
 terramate create path/to/stack
 ```
 
-Complete Example:
+### Import Terraform Root Modules
+
+If you already have an existing Terraform setup in your repository and want to initialize all directories containing Terraform configurations as Terramate Stacks, you can use `--all-terraform` option to automatically detect those stacks and initialize them.
+
+Terramate detects all Terraform directories that contain a `backend {}` configuration as a stack for you.
```

Comment:
I don't know if it's that important to be accurate here but we detect `terraform.backend` block or `provider` block.

Reply:
will add in a next iteration. thx for clarifying.